events.py

```diff
@@ -221,6 +221,9 @@ def generate_ical_file(generator):
     DEFAULT_LANG = generator.settings['DEFAULT_LANG']
     curr_events = events if not localized_events else localized_events[DEFAULT_LANG]
 
+    filtered_list = list(filter(lambda x: x.event_plugin_data["dtstart"] >= datetime.now(), curr_events))
+    curr_events = filtered_list
+
     for e in curr_events:
```

Comment:
```suggestion
    filtered_list = filter(lambda x: x.event_plugin_data["dtstart"] >= datetime.now(), curr_events)

    for e in filtered_list:
```

Der Iterator, den `filter(...)` zurückgibt, sollte hier genügen. Ein zusätzliches `list(...)` erzwingt unnötig eine weitere Kopie der kompletten `events` / `localized_events[DEFAULT_LANG]` Liste im RAM.